### PR TITLE
Add multiple buyers per contract extension

### PR DESCRIPTION
**Pull request by transpresupuestaria**

### CODE REVIEW

File: multiple_buyers/entry.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[{
+  "name": {
+    "en" : "Multiple_Buyers_Extension"
```
Comment (timgdavies) [inline multiple_buyers/entry.json]: The underscore is not needed in the name here. 

Suggested name for this extension is 'Multiple buyers per contract'

File: multiple_buyers/entry.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  },
+  "description": {
+    "en": "This extension provides a way to provide multiple buyers per contract."
```
Comment (timgdavies) [inline multiple_buyers/entry.json]: I would suggest updating the description to:

"Adds a buyers array to each contract, allowing one or more buyers to be specified for each individual contract within an overall contracting process."

**Comment by jpmckinney** [comment]
I've opened a few pull requests, one of which must be merged before adding to the extension registry: https://github.com/transpresupuestaria/ocds_multiple_buyers_extension/pull/3

The extension registry format changed slightly. There are new instructions in the readme, but we would be happy to facilitate the addition of this extension once the mentioned PR is merged.

**Comment by jpmckinney** [comment]
In the meantime, we will close this issue, as a new PR (to match the new registry format) will be needed.

**Comment by jpmckinney** [comment]
Linking to discussion in https://github.com/open-contracting/ocds-extensions/issues/86

**Pull request closed without merging**